Source/Plugins/Core/com.equella.core/js/tsrc/fileuploader/InlineFileUploader.tsx

```diff
@@ -131,11 +132,22 @@ export const InlineFileUploader = ({
   strings,
   reloadState,
 }: InlineFileUploaderProps) => {
+  const initialiseEntry = (
+    entry: AjaxFileEntry,
+    indented: boolean
+  ): UploadedFile => ({
+    fileEntry: entry,
+    status: "uploaded",
+    indented: indented,
+  });
+
   const [uploadedFiles, setUploadedFiles] = useState<UploadedFile[]>(
-    entries.map((entry) => ({
-      fileEntry: entry,
-      status: "uploaded",
-    }))
+    entries.flatMap((entry) => {
+      const withChildren: UploadedFile[] = [];
+      return withChildren
+        .concat(initialiseEntry(entry, false))
+        .concat(entry.children.map((e) => initialiseEntry(e, true)));
+    })
```

Comment:
I wonder if we could make it simpler with just:

```suggestion
    entries.flatMap((entry) => [
      initialiseEntry(entry, false),
      ...(entry.children.map((e) => initialiseEntry(e, true)))
    ])
```